Heapsort

```diff
@@ -0,0 +1,79 @@
+void heapsort(int A[],int n);
+void Build_Heap (int A[],int n);
+void Heapify(int A[],int i, int n);
+
+
+void main()
+
+{
+	int x;
+	int A[10]={9,8,2,3,4,5,1,2,3,1};
+	heapsort(A,10);
+
+	for(x=0;x<10;x++)
+	{
+	printf("Heap Sort %d\n",A[x]);
+	}
+
+
+
+}
+
+void heapsort(int A[],int n)
```

Comment:
Could you name the variables more verbosely. Single letter variables llike A and n are very difficult to track after some time.